wolox-books/src/app/screens/modal-car/modal-car.component.html

```diff
@@ -0,0 +1,10 @@
+<div>
+  <h2 class="title-modal">Tu carrito de compras</h2>
+  <div class="content-books-car" *ngFor="let item of books | async; let i = index">
+    <span class="text-modal">{{ item.title }} - {{ item.author }} </span>
```

Comment:
what happens if the text is too long for the container width? What could you use to have a default text length? SPOILER: |